packages/uikit-react-native-foundation/src/ui/GroupChannelMessage/MessageContainer.tsx

```diff
@@ -34,43 +35,48 @@ MessageContainer.Incoming = function MessageContainerIncoming({
   const color = colors.ui.groupChannelMessage.incoming;
 
   return (
-    <Box flexDirection={'row'} justifyContent={'flex-start'} alignItems={'flex-end'}>
-      <Box width={26} marginRight={12}>
-        {(message.isFileMessage() || message.isUserMessage()) && !groupedWithNext && (
-          <Pressable onPress={onPressAvatar}>
-            <Avatar size={26} uri={message.sender?.profileUrl} />
-          </Pressable>
-        )}
-      </Box>
-      <Box flexShrink={1}>
-        {parentMessage}
-        {!groupedWithPrev && !message.parentMessage && (
-          <Box marginLeft={12} marginBottom={4}>
-            {(message.isFileMessage() || message.isUserMessage()) && (
-              <Text caption1 color={color.enabled.textSenderName} numberOfLines={1}>
-                {strings?.senderName ?? message.sender.nickname}
-              </Text>
-            )}
-          </Box>
-        )}
-
-        <Box flexDirection={'row'} alignItems={'flex-end'}>
-          <Box style={styles.bubble}>{children}</Box>
-          {!groupedWithNext && (
-            <Box marginLeft={4}>
-              <Text caption4 color={color.enabled.textTime}>
-                {strings?.sentDate ?? getMessageTimeFormat(new Date(message.createdAt))}
-              </Text>
+    <Box flexDirection={'column'} justifyContent={'flex-start'} alignItems={'flex-start'}>
+      <Box flexDirection={'row'} justifyContent={'flex-start'} alignItems={'flex-end'}>
+        <Box width={26} marginRight={12}>
+          {(message.isFileMessage() || message.isUserMessage()) && !groupedWithNext && (
+            <Pressable onPress={onPressAvatar}>
+              <Avatar size={26} uri={message.sender?.profileUrl} />
+            </Pressable>
+          )}
+        </Box>
+        <Box flexShrink={1}>
+          {parentMessage}
+          {!groupedWithPrev && !message.parentMessage && (
+            <Box marginLeft={12} marginBottom={4}>
+              {(message.isFileMessage() || message.isUserMessage()) && (
+                <Text caption1 color={color.enabled.textSenderName} numberOfLines={1}>
+                  {strings?.senderName ?? message.sender.nickname}
+                </Text>
+              )}
             </Box>
           )}
+          <Box flexDirection={'row'} alignItems={'flex-end'}>
+            <Box style={styles.bubble}>{children}</Box>
+            {!groupedWithNext && (
+              <Box marginLeft={4}>
+                <Text caption4 color={color.enabled.textTime}>
+                  {strings?.sentDate ?? getMessageTimeFormat(new Date(message.createdAt))}
+                </Text>
+              </Box>
+            )}
+          </Box>
         </Box>
       </Box>
+      <Box marginLeft={40} marginTop={4} flexDirection={'row'} height={20}>
+        {replyInfo}
+      </Box>
```

Comment:
In the case of general messages, there is no replyInfo.
However, since it occupies space, it creates an empty space in the UI.
It seems like we need conditional rendering

![image](https://github.com/sendbird/sendbird-uikit-react-native/assets/26326015/06d59dfd-9673-4746-98f3-0fe80407a6fc)

### Conditional rendering
```tsx
{ condition ? <Component /> : null }

{ !!condition && <Component /> }
{ Boolean(condition) && <Component /> }
```

When using `condition && <Component />`, if the condition is `0` or an empty string(`""`), `0` or the empty string gets rendered, causing an error that says `text must be nested within a Text component`.
Therefore, when using the `&&` operator, you should also use `!!` or `Boolean` to ensure the condition is properly evaluated as a boolean.

Reply:
관련코드 GroupChannelMessageReplyInfo 에서 처리하도록 옮기고 수정하였습니다.